### PR TITLE
ipatests: Test for ipa-nis-manage CLI tool

### PR DESCRIPTION
**Pull request by menonsudhir**
The testcases check the various options of ipa-nis-manage CLI tool as below
    ipa-nis-mange enable
    ipa-nis-manage disable
    Enabling NIS plugin with invalid admin password

### CODE REVIEW

File: ipatests/test_integration/test_adtrust_install.py [hunk 1]
```diff
@@ -266,3 +270,81 @@ def test_ipa_user_s4u2self_pac(self):
...
+        )
+        assert result.returncode == 4
+        assert status_msg in result.stdout_text
+
+    def test_ipa_nis_manage_enable_incorrect_password(self):
+        """
+        This testcase checks if ipa-nis-manage enable
+        command throws error on console for invalid DS admin password
+        """
+        msg = "Insufficient access:  Invalid credentials"
```
Comment (mrizwan93) [inline ipatests/test_integration/test_adtrust_install.py:343]:
```
AssertionError: assert 'Insufficient access:  Invalid credentials' in 'Insufficient access: Invalid credentials Invalid credentials\n'
```
There is extra space in both the string.

File: ipatests/test_integration/test_adtrust_install.py [hunk 2]
```diff
@@ -266,3 +270,81 @@ def test_ipa_user_s4u2self_pac(self):
...
+        )
+        assert console_msg in result.stdout_text
+        self.master.run_command(["systemctl", "restart", dirsrv_service])
+        time.sleep(DIRSRV_SLEEP)
+        self.master.run_command(["systemctl", "restart", "rpcbind"])
+        result = self.master.run_command(
+            ["ipa-nis-manage", "status"],
+            stdin_text=self.master.config.admin_password,
+        )
+        assert status_msg in result.stdout_text
```
Comment (rcritten) [inline ipatests/test_integration/test_adtrust_install.py:312]: If you want to not trust the command output you can check directly whether the plugin is enabled by pulling the entry:

DN(('cn', 'NIS Server'), ('cn', 'plugins'), ('cn', 'config'))

and ensuring that nsslapd-pluginenabled.lower() == 'on'

Similar with disable.

**Comment by rcritten** [comment]
Is there a reason this is included in the AD tests? test_commands might be more appropriate.

File: ipatests/test_integration/test_adtrust_install.py (hunk 2 above)
Comment (menonsudhir) [reply ipatests/test_integration/test_adtrust_install.py]: Added the change to pull the entry when plugin is enabled/disabled.

**Comment by menonsudhir** [comment]
> Is there a reason this is included in the AD tests? test_commands might be more appropriate.
Added the testcase to test_commands

File: ipatests/prci_definitions/temp_commit.yaml [hunk 3]
```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_adtrust_install.py::TestIpaNisManage
```
Comment (rcritten) [inline ipatests/prci_definitions/temp_commit.yaml:71]: You need to update the temp commit with the new test location

Comment (menonsudhir) [reply ipatests/prci_definitions/temp_commit.yaml]: Updated the temp commit with correct test file name.

**Comment by rcritten** [comment]
LGTM, please remove the temporary commit.

**Comment by menonsudhir** [comment]
@rcritten could you please ACK.

**Comment by rcritten** [comment]
master:

* 6ff31dbf55cfea509289c72c6f97f945fb934f6c ipatests: Test for ipa-nis-manage CLI tool.

**Pull request closed without merging**